src/utils/slangHelper.ts

```diff
@@ -34,13 +24,29 @@ declare global {
  *   which REPL the value shows up in.
  */
 function display(value: Value, workspaceLocation: any) {
-  const output = toString(value)
+  const output = stringify(value)
   // TODO in 2019: fix this hack
   if (typeof (window as any).__REDUX_STORE__ !== 'undefined') {
     ;(window as any).__REDUX_STORE__.dispatch(handleConsoleLog(output, workspaceLocation))
   }
+  return value
+}
+
+/**
+ * Function that takes a value and displays it in the interpreter.
+ * The value is displayed however native JS would convert it to a string.
+ * e.g. [1, 2, 3] would be displayed as 1,2,3.
+ * An action is dispatched using the redux store reference
+ * within the global window object.
+ *
+ * @param value the value to be displayed
+ * @param workspaceLocation used to determine
+ *   which REPL the value shows up in.
+ */
+function rawDisplay(value: Value, workspaceLocation: any) {
+  display(String(value), workspaceLocation)
```

Comment:
It should be the other way around: display should call rawDisplay. I'm not even sure this works correctly at the moment (seems like quotes will get escaped).

Reply:
Whoops yeah, they're flipped. I'll just swap them back. Yeah display will call stringify first and then call rawdisplay, escaping the strings.